lib/stm32/h7/pwr.c

```diff
@@ -19,8 +13,7 @@
 /*
  * This file is part of the libopencm3 project.
  *
- * Copyright (C) 2011 Stephen Caudle <scaudle@doceme.com>
- * Copyright (C) 2017 Matthew Lai <m@matthewlai.ca>
+ * Copyright (C) 2019 Brian Viele <vielster@allocor.tech>
```

Comment:
it's generally considered bad form to drop other people's copyright statements...  I'd agree that at least scaudle probably had nothing in this file anyway, but mlai did the f7 code this was based on iirc?

Reply:
Re-added...I'm not sure if there is a single line that was retained across the two, but I also don't have any issue with sharing this copyright.